frontend/src/modules/entitieslist/components/Entity.tsx

```diff
@@ -122,21 +155,39 @@ export default class Entity extends React.Component<Props> {
 
         const classChecked = checkedForBatchEditing ? 'checked' : '';
 
+        const classSibling = entity.isSibling ? 'sibling' : '';
         return (
             <li
-                className={`entity ${this.status} ${classSelected} ${classBatchEditable} ${classChecked}`}
+                className={`entity ${this.status} ${classSelected} ${classBatchEditable} ${classChecked} ${classSibling}`}
                 onClick={this.selectEntity}
             >
                 <span
                     className='status fa'
                     onClick={this.toggleForBatchEditing}
                 />
+                {classSelected && !classSibling ? (
+                    <div>
+                        {parameters.search || this.areFiltersApplied() ? (
+                            <Localized id='entitieslist-Entity--sibling-strings-title'>
+                                {!this.state.siblingsAreActive && (
```

Comment:
Is there are reason to have this condition within the `<Localized>` element instead of putting it in line 170?

Logically all three of them serve the same purpose - showing or hiding the same button. So they should be all together, and if it make code easier to read, you could also factor them out into a function (e.g. `showSiblingEntitiesButton`).

Reply:
there was no reason for it, and you are correct; it does do the same thing to show the button
here is the refactor in a function 
```
showSiblingEntitiesButton = () => {
        const isSearched = this.props.parameters.search;
        const areFiltersApplied = this.areFiltersApplied();
        const areSiblingsActive = !this.state.areSiblingsActive;

        return (isSearched || areFiltersApplied) && areSiblingsActive;
    };
    
    {this.showSiblingEntitiesButton() && (
                            <Localized id='entitieslist-Entity--sibling-strings-title'>
                                <i
                                    className={
                                        'sibling-entities-icon fas fa-expand-arrows-alt'
                                    }
                                    title='Click to reveal sibling strings'
                                    onClick={this.getSiblingEntities}
                                ></i>
                            </Localized>
                        )} 
```

Reply:
Let me know if the way I have the function is readable.

Reply:
LGTM.

Reply:
I think I'm getting really old, it took me way to long to decipher what this meant 😭

Reply:
Hehe :) Ok, let's make these two changes and we ship.